subsys/bluetooth/controller/ll_sw/nordic/lll/lll_df.c

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <stdint.h>
+#include "hal/radio_df.h"
+#include "lll_df.h"
+
+/* @brief Function performs common steps for initialization and reset
+ * of Direction Finding LLL module.
+ *
+ * @return	Zero in case of success, other value in case of failure.
+ */
+static int init_reset(void);
+
+/* @brief Function performs Direction Finding initialization
+ *
+ * @return	Zero in case of success, other value in case of failure.
+ */
+int lll_df_init(void)
+{
+	int err;
+
+	err = init_reset();
+	if (err) {
+		return err;
+	}
+
+	return 0;
+}
+
+
+/* @brief Function performs Direction Finding reset
+ *
+ * @return	Zero in case of success, other value in case of failure.
+ */
+int lll_df_reset(void)
+{
+	int err;
+
+	err = init_reset();
+	if (err) {
+		return err;
+	}
+
+	return 0;
+}
+
+static int init_reset(void)
+{
+	int err;
+
+	err = radio_df_ant_configure();
```

Comment:
if the configuration will not be changed after power up, then place `radio_df_ant_configure` only in `lll_df_init`

Reply:
At the moment it will be not changed after power up.
I did following change:
- removed `init_reset()` because it is no longer needed,
- `radio_df_ant_configure` is called directly in `lll_df_init`
- added `lll_df_reset` to `perform_lll_reset`.

Reply:
Please keep the `init_reset()` as a template for future implementation, the compiler removes it if its empty. Otherwise, changes in this PR looks good to me.

Reply:
`init_reset()` restored